tools/jenkins/run_interop.sh

```diff
@@ -37,3 +37,5 @@ export LANG=en_US.UTF-8
 cd $(dirname $0)/../..
 
 tools/run_tests/run_interop_tests.py -l all -s all --cloud_to_prod --cloud_to_prod_auth --use_docker --http2_interop -t -j 12 $@ || true
+tools/run_tests/run_interop_tests.py -l java --use_docker --http2_badserver_interop $@ || true
```

Comment:
Adele is right here, the way it's done here is broken, the report will get overwritten.